src/test/PDFTest/AnnotationPDFServiceTest.java

```diff
@@ -270,6 +380,7 @@ public void fillApplicationQuestionsSS5Test() throws IOException, GeneralSecurit
     body = new JSONObject();
     body.put("applicationId", fileId);
     body.put("formAnswers", formAnswers);
+    body.put("clientUsername", clientUsername);
```

Comment:
you put the client username field here but you should assert this is there in the final PDF signature metadata